docs/workflow_solutions/creating_sandbox_apps.md

```diff
@@ -0,0 +1,120 @@
+
+1. [Setting up Sandbox for your Cheaha Account](#setting-up-sandbox-for-your-cheaha-account)
+1. [Building your first dev app](#building-your-first-dev-app)
+   - [FSL](#fsl)
+1. [Launching your Sandbox App](#launching-your-sandbox-app)
+1. [More info](#more-info)
+
+## Setting up Sandbox for your Cheaha Account
+
+The following steps would guide you to set up Sandbox for your Cheaha account.
+
+You need to create a dev folder in your `$USER/ondemand` folder by following these steps:
+
+1. Create a HPC Desktop Interactive Job on Cheaha. There is a detailed guide [here](../cheaha/open_ondemand/ood_layout.md#creating-an-interactive-job) on how to do this.
+
+1. Access the terminal within your HPC Desktop Job. There is a guide on how to access the terminal [here](../cheaha/tutorial/pytorch_tensorflow.md#installing-anaconda-environments-using-the-terminal)
+
+    ![Terminal button from Interactive sessions page on Cheaha](../cheaha/tutorial/images/cheaha_shell_button.png)
+
+1. Create the folder within the terminal using the command `mkdir -p $USER_DATA/ondemand/dev`.
+
+1. Go to the [Cheaha](https://rc.uab.edu) Open OnDemand homepage.
+
+1. When logged in, click on the "Help" button, you should see the button in the top-right corner of the landing page of Cheaha, click on "Restart Web Server".
+
+    ![Help button menu options](images/cheaha_help_button.png)
+
+1. When the page reloads, you should see a new "Develop" tab have successfully enabled Sandbox for your environment, and you should see a new "Develop" tab.
+
+    ![Develop button on Cheaha](images/cheaha_develop_button.png)
+
+<!-- markdownlint-disable MD046 -->
+!!! note
+
+    This is a one time setup, and would not need to be repeated for subsequent apps you would like to create.
+<!-- markdownlint-enable MD046 -->
+
+## Building your first dev app
+
+Now that Sandbox is set up on your account, we can create an App. The instructions below detail how to create your first example dev app via a container:
+
+### FSL
+
+This example using the FSL (FMRIB Software Library) shows how to create a simple Sandbox App on Cheaha.
+
+1. Under "Develop" tab , click on "My SandBox Apps (Development)"
+
+    ![Develop button options on Cheaha](images/cheaha_develop_sandbox.png)
+
+1. Clicking the "New App" button, should direct you to a new page giving you an option to Clone an Existing App.
+
+    ![New App button to Clone an Existing App](images/sandbox_new_app.png)
+
+1. Now we are going to import an already existing git repo for an app, by clicking on the "Clone Existing App". A form (shown below) requiring your input will display. For the FSL App we will use a template we already created (you can use this template to prepare your app as well).
+
+    ![Form for new Sandbox App](images/sandbox_new_app_form.png)
+
+    We will add the following values in the form:
+
+    **Directory Name:** FSL
+
+    **Git Remote:** <https://gitlab.rc.uab.edu/rc/bc_uab_igv.git> and then click "**Submit**"
```

Comment:
Since you are asking someone to enter text, use backticks on the URL to make it a verbatim text span, instead of a hyperlink.

Make the "Click submit" not part of this unordered list.